lib/rules/no-unexpected-multiline.js

```diff
@@ -82,6 +82,10 @@ module.exports = {
 
             "BinaryExpression[operator='/'] > BinaryExpression[operator='/']"(node) {
```

Comment:
This is a [selector](http://eslint.org/docs/developer-guide/selectors) -- the function will get called whenever there is a division expression which is a child of another division expression. It's designed to work when the child expression is in the *left* part of the division expression:

```js
foo
/bar/g

// this is actually (foo / bar) / g, and the `node` parameter in the function is `foo / bar`.
```

In the case where `node` is on the left, there will always be a `/` token after `node` because the parent is also a division expression. However, when `node` is on the right, there isn't always a `/` token after the child, which is why `sourceCode.getTokenAfter` returned null and an error was thrown.

As-is, I'm not sure your fix is correct, because there might be another `/` token later in an unrelated part of the program:

```js
5 / (5
/ 5);

var foo = bar / baz; // `secondSlash` for the division above will be the `/` token here, which is unexpected.
```

Instead, I think the best way to fix this is to make sure the function only gets called when the child division expression is on the *left* of its parent. This can be done by adding a field to the selector:

```js
"BinaryExpression[operator='/'] > BinaryExpression[operator='/'].left"(node) {
    // ...
}
```